README.md

```diff
@@ -3,13 +3,25 @@
 
 This repository contains instructions to build modern Cloud-based Air Traffic Control using IBM Bluemix.
 
-Air Traffic Control receives flight information from Raspberry Pi powered ADS-B Ground Stations with Software Defined Radio(SDR) to receive ADS-B messages directly from commercial flights and publish MQTT messages to the IBM IoT Platform running in IBM Bluemix. It also supports a Swift-based iOS app to track flights using Augmented Reality toolkit by receiving MQTT messages from the IoT Platform.
+The Air Traffic Control application receives flight information from a Raspberry Pi powered ADS-B Ground Stations with Software Defined Radio(SDR) to receive ADS-B messages directly from commercial flights and publish MQTT messages to the IBM IoT Platform running in IBM Bluemix. It also supports a Swift-based iOS app to track flights using the Augmented Reality toolkit by receiving MQTT messages from the IoT Platform. The app will display all the flights traveling point to point within the range of the receiver.
 
```

Comment:
I don't think `application` is the right word here to describe `Air Traffic Control`. You should just call it `Air Traffic Control` service or platform.